yarn.lock

```diff
@@ -7012,6 +7024,17 @@ react-virtualized-auto-sizer@^1.0.2:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/react-virtualized-auto-sizer/-/react-virtualized-auto-sizer-1.0.2.tgz#a61dd4f756458bbf63bd895a92379f9b70f803bd"
 
+react-virtualized@^9.21.0:
```

Comment:
should this be reverted if we are not adding react-virtualized?

Reply:
yep